src/server/debug/server/server.go

```diff
@@ -361,11 +386,11 @@ func (s *debugServer) Profile(request *debug.ProfileRequest, server debug.Debug_
 		pachClient,
 		server,
 		request.Filter,
-		collectProfileFunc(request.Profile),
-		nil,
-		nil,
-		redirectProfileFunc(request.Profile),
-		collectProfileFunc(request.Profile),
+		collectProfileFunc(request.Profile),  // collectPachd
+		nil,                                  // collectPipeline
```

Comment:
nit: for annotating naked parameters, I kinda prefer C-style comments.

```suggestion
		nil /* collectPipeline */,
```

But I can see an argument in favor of aligning the comments like this since every parameter is on a new line.

Reply:
I don't really have a preference.